docker/Dockerfile

```diff
@@ -41,13 +41,17 @@ RUN python3 -m venv /home/venv
 
 ENV PATH="/home/venv/bin:$PATH"
 
-RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 1
-RUN update-alternatives --install /usr/local/bin/pip pip /usr/local/bin/pip3 1
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 1 \
+    && update-alternatives --install /usr/local/bin/pip pip /usr/local/bin/pip3 1
 
-# This is only useful for cuda env 
+# This is only useful for cuda env
 RUN export USE_CUDA=1
 
-RUN pip install --no-cache-dir torch torchvision torchtext torchserve torch-model-archiver
+RUN if [ "$BASE_IMAGE" != "${BASE_IMAGE#*cuda:10.1}" ]; then \
+    pip install --no-cache-dir torch==1.6.0+cu101 torchvision==0.7.0+cu101 -f https://download.pytorch.org/whl/torch_stable.html; \
```

Comment:
@harshbafna Shouldn't this be in the else part based on the check?

Also please have the torchvision + cuda version a variable so that we don't have to keep changing the Dockerfile with each release

Reply:
@harshbafna ping

Reply:
@harshbafna Any update on this?

Reply:
@chauhang: This has been taken care of. @jeremiahschung has implemented these changes. Could you please review it once again?